Feature/Enhancement: Set base for all selected watches & update misc typos, update appveyor and readme

dreamsyntax opened the pull request:
Adds new feature:
* All selected items can have base set with a single click (ex: Octal, Binary, Hex, Decimal)

Corrects Typos:
* alignement -> alignment (inc function signatures)

Pipeline Update:
* Update to Visual Studio 2019 (for correct Win 10 SDK)
* Update README

aldelaro5 commented:
I'll need to test this, but this looks good for now.

Sorry if I take a while btw, this project isn't being maintained much for various reasons.

dreamsyntax commented:
After discussion (unrelated to this) on a separate PR in the Dolphin repo,
> Cu&t -> &Cut

Was actually intentionally Cu&t for hotkey purposes. Reverted my incorrect 'typo correction' change regarding Cu&t and updated my description
